whitelabelco/sofo-foods
# update history page with missing gallery

### Description 💻

Resolves: #134 

### Implementation 💡

updates the History page with missing gallery asset `1964-1985`
[comps](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=153-1437&node-type=frame&t=znA73cuVrXnVoD7U-0)


#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- scroll down and click on "Our History"
- verify `1964-1985` gallery is update and matches comps

### Screenshots/Demos 📷


### Checklist ✅
